classes/dejitter_group/dejitter_group.gd

```diff
@@ -18,6 +20,7 @@ func _physics_process(_delta):
 			offset_vec = Vector2.ZERO
 		else:
 			offset_vec = carrier_node.texture.get_size().posmod(2) / 2
+	offset_vec += dejitter_position
```

Comment:
A comment here might help. Not sure what this does.

Reply:
I can't remember either. Time for some code archaeology ⛏️

Reply:
Done.

...actually, I went ape and commented the entirety of `DejitterGroup`, including a brief docstring.